ReactWindows/ReactNative.Shared/UIManager/AccessibleCanvas.cs

```diff
@@ -0,0 +1,30 @@
+#if WINDOWS_UWP
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Automation.Peers;
+#else
+using System.Windows.Controls;
+using System.Windows.Automation.Peers;
+#endif
+
+namespace ReactNative.UIManager
+{
+    /// <summary>
+    /// Canvas with ReactXP/ReactNative accessibility properties support. 
+    /// </summary>
+    public class AccessibleCanvas : Canvas, IAccessible
+    {
+        /// <inheritdoc />
```

Comment:
>/// <inheritdoc /> [](start = 8, length = 18)

I've never seen this before. Does VS support this?

Reply:
Yes, it's valid xmldoc element to use parent's xmldoc. I added it because the c# project already have a rule enabled that gives warning for a public member or type without xmldoc.